server/servers/deploy/src/test/scala/com/prisma/deploy/specutils/DeployTestServer.scala

```diff
@@ -114,10 +116,38 @@ case class DeployTestServer()(implicit dependencies: DeployDependencies) extends
         )
         .recover {
           case error: QueryAnalysisError => error.resolveError
+          case e                         => sys.error(e.toString)
         },
       Duration.Inf
     )
     println("Request Result: " + result)
     result
   }
+
+  /**
+    * Deploy a schema and return a project
+    */
+  def deploySchema(project: Project, schema: String): Future[Option[Project]] = {
```

Comment:
This is some duplication of what we already have in `DeploySpecBase.setupProject`. Can you add an `addProject` here and call those methods from `DeploySpecBase` please?

Reply:
Done